tests/test_data/projects/napp_with_annotation_processor/snowflake.yml

```diff
@@ -0,0 +1,25 @@
+definition_version: 1
+native_app:
+  name: myapp
+
+  # the following shows how to properly use a quoted identifier in YAML
+  source_stage: >-
+    "MySourceSchema"."SRC_Stage"
+
+  artifacts:
+    - setup.sql
+    - app/README.md
+    - src: app/streamlit/*.py
+      dest: ui/
+      processors:
+        - snowpark # str instead of dict
+        - name: snowpark # without virtual_env
+        - name: snowpark # with virtual_env
```

Comment:
maybe vary them a little? Eventually we'll likely consider this an invalid PDF due to repetition